### PR TITLE
feat: add formulas 8.73, 8.74 and 8.75 from FprEN 1992-1-1:2023 clause 8.2.6

### PR DESCRIPTION
**Pull request by egarciamendez**
Opens clause 8.2.6, shear at interfaces, of FprEN 1992-1-1:2023 (E): the verification itself and the two routes to the shear stress that goes into it. Pages 130 and 131.

- Formula (8.73), `Form8Dot73CheckShearStressAtInterface`, art. 8.2.6(3). The verification of the shear stress at the interface against its resistance.
- Formula (8.74), `Form8Dot74DesignShearStressAtInterface`, art. 8.2.6(4). The shear stress from a shear force acting parallel to the interface.
- Formula (8.75), `Form8Dot75LongitudinalShearStressDueToCompositeAction`, art. 8.2.6(4). The longitudinal shear stress between concrete interfaces due to composite action. It sits in the same paragraph as (8.74), on the following page, and is a second route to the same symbol rather than a refinement of the first.

Both (8.74) and (8.75) return the same printed symbol from different actions, so the two shear forces are spelled out in each docstring: `v_edi` is the force parallel to the interface in (8.74), `v_ed` the force perpendicular to it in (8.75). Lowercasing collapses `V_Edi` and `V_Ed` onto names that differ by one letter, which is exactly the kind of pair that gets mixed up silently.

The resistance `tau_rdi` of (8.73) is an input. Formulas (8.76) and (8.77), which produce it, are the next clause paragraphs and are not in this pull request; (8.76) also needs Table 8.2 for its roughness factors.

### Notes

- (8.73) is printed with `≤`, so equality has to pass. The test covers the boundary explicitly.
- The parameter `v_edi` of (8.74) is a force while a lowercase `v` normally denotes a stress in this document. The type alias `N` and the docstring carry the distinction; renaming it would break the symbol correspondence with the printed `V_Edi`.
- The standard itself calls `V_Edi` parallel to the interface in (8.74) and `V_Ed` perpendicular to it in (8.75). That wording is reproduced as printed, since it is the standard's own inconsistency and not one to be repaired here.

Contributes to #1083.

## Formulas as implemented

**Formula (8.73)**, `Form8Dot73CheckShearStressAtInterface`

`equation`

$$
CHECK \to \tau_{Edi} \leq \tau_{Rdi}
$$

`complete`

$$
CHECK \to \tau_{Edi} \leq \tau_{Rdi} \to 0.800 \leq 1.200 \to OK
$$

`complete_with_units`

$$
CHECK \to \tau_{Edi} \leq \tau_{Rdi} \to 0.800 \ MPa \leq 1.200 \ MPa \to OK
$$

**Formula (8.74)**, `Form8Dot74DesignShearStressAtInterface`

`equation`

$$
\tau_{Edi} = \frac{V_{Edi}}{A_i}
$$

`complete`

$$
\tau_{Edi} = \frac{V_{Edi}}{A_i} = \frac{200000.000}{250000.000} = 0.800 \ MPa
$$

`complete_with_units`

$$
\tau_{Edi} = \frac{V_{Edi}}{A_i} = \frac{200000.000 \ N}{250000.000 \ mm^2} = 0.800 \ MPa
$$

**Formula (8.75)**, `Form8Dot75LongitudinalShearStressDueToCompositeAction`

`equation`

$$
\tau_{Edi} = \frac{\beta_{new} \cdot V_{Ed}}{z \cdot b_i}
$$

`complete`

$$
\tau_{Edi} = \frac{\beta_{new} \cdot V_{Ed}}{z \cdot b_i} = \frac{0.450 \cdot 500000.000}{450.000 \cdot 500.000} = 1.000 \ MPa
$$

`complete_with_units`

$$
\tau_{Edi} = \frac{\beta_{new} \cdot V_{Ed}}{z \cdot b_i} = \frac{0.450 \cdot 500000.000 \ N}{450.000 \ mm \cdot 500.000 \ mm} = 1.000 \ MPa
$$

## Summary by CodeRabbit

* **New Features**
  * Added Eurocode calculations for interface shear-stress verification, design shear stress, and longitudinal shear stress from composite action.
  * Added numerical results and formatted LaTeX representations with unit-aware output.
* **Documentation**
  * Marked formulas 8.73, 8.74, and 8.75 as implemented and linked them to their calculations.
* **Tests**
  * Added coverage for calculations, boundary conditions, input validation, failure cases, and LaTeX output formats.

### CODE REVIEW

File: blueprints/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/formula_8_75.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        beta_new: DIMENSIONLESS,
+        v_ed: N,
+        z: MM,
+        b_i: MM,
+    ) -> MPA:
+        """Evaluates the formula, for more information see the __init__ method."""
+        raise_if_negative(beta_new=beta_new, v_ed=v_ed)
+        raise_if_less_or_equal_to_zero(z=z, b_i=b_i)
+
+        return beta_new * v_ed / (z * b_i)
```
[the remark this answers is not in the document]
Comment (egarciamendez) [reply blueprints/codes/eurocode/fpr_en_1992_1_1_2023/chapter_8_ultimate_limit_states/formula_8_75.py:59]: No range is printed for `beta_new` at (8.75). This track only guards what the standard states, so not added.

**Review by GerjanDorgelo — APPROVED** [review]
